src/LndClient/Data/ListChannels.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+
+module LndClient.Data.ListChannels
+  ( ListChannelsRequest (..),
+    ListChannelsResponse (..),
+    Channel (..),
+  )
+where
+
+import qualified Data.Text.Lazy as TL (Text)
+import LndClient.Import
+import qualified LndGrpc as GRPC
+
+data ListChannelsRequest
+  = ListChannelsRequest
+      { activeOnly :: Bool,
+        inactiveOnly :: Bool,
+        publicOnly :: Bool,
+        privateOnly :: Bool,
+        peer :: Maybe ByteString
+      }
+  deriving (Generic, Show)
+
+data ListChannelsResponse
+  = ListChannelsResponse
+      { channels :: [Channel]
+      }
+  deriving (Generic, Show)
+
+data Channel
+  = Channel
+      { remotePubkey :: TL.Text,
+        channelPoint :: TL.Text
```

Comment:
we need to use ChannelPoint type there, to make types and interface more logical

Reply:
Added TODO